cli/tauri-bundler/src/bundle/wix.rs

```diff
@@ -539,6 +566,105 @@ pub fn build_wix_app_installer(
   Ok(target)
 }
 
+// sign code forked from https://github.com/forbjok/rust-codesign
```

Comment:
Just like with MacOS, I think we should move this to a separate file for simplicity and to avoid a nearly 1000-line file.

Reply:
> I haven't tested it, but it looks like it would work 😂. I just think we should move the signing logic into separate files since the build files are so long now.

it doesn't work lol i've tested it last year.